### PR TITLE
add support for Gradle projects

### PR DESCRIPTION
**Pull request by daspilker**

### CODE REVIEW

**Comment by daspilker** [comment]
a Gradle project can be tested here: https://ci.jenkins.io/job/Plugins/job/job-dsl-plugin/view/Pull%20Requests/job/PR-994/

File: vars/buildPlugin.groovy [hunk 1]
```diff
@@ -34,47 +36,73 @@ def call(Map params = [:]) {
...
+                            } else {
+                                List<String> gradleOptions = [
+                                        '--no-daemon',
+                                        'cleanTest',
+                                        'build',
+                                ]
+                                command = "gradlew ${gradleOptions.join(' ')}"
+                                if (isUnix()) {
+                                    command = "./" + command
+                                }
```
Comment (orrc) [inline vars/buildPlugin.groovy]: What about Windows (`gradlew.bat`)?

Comment (daspilker) [inline vars/buildPlugin.groovy]: It's not necessary to add the extension on Windows.

Comment (orrc) [inline vars/buildPlugin.groovy]: Good point! But in any case, sorry — I misread.  This is just adding the `./` prefix for Unix, not actually *running* the command…

**Comment by rtyler** [comment]
@daspilker thanks for taking the time to do this :fireworks:

**Pull request merged**